README.md

```diff
@@ -2,29 +2,30 @@
 
 ![Calva logo](assets/calva-64h.png)
 
-_Distilled from [CIDER](https://cider.mx/)*._
+_Distilled from [CIDER](https://cider.mx/)\*._
 
 Welcome to [Calva](https://marketplace.visualstudio.com/items?itemName=betterthantomorrow.calva), an integrated, REPL powered, development environment for enjoyable and productive [Clojure](https://clojure.org) and [ClojureScript](https://clojurescript.org) programming in [Visual Studio Code](https://code.visualstudio.com). It includes:
 
-* inline [code evaluation](https://calva.io/evaluation/)
-* [structural editing](https://calva.io/paredit/)
-* code [formatting](https://calva.io/formatting/)
-* code navigation
-* a [debugger](https://calva.io/debugger/)
-* [linting](https://calva.io/linting/)
-* syntax highlighting
-* Clojure aware rainbow brackets
-* a [test runner](https://calva.io/test-runner/)
-* refactoring support
-* and much more.
```

Comment:
I don't fancy the bulk change to the README about the bullets, though. Generally I use asterisk  as bullets and the Markdown spec clearly allows it. Can we discuss it separately?

Reply:
Interesting. I wonder if we may have different prettier settings. Let me check if I have something in my user settings about it...

Reply:
It happens with the prettier extension enabled if I run "Format Document" on the readme and have prettier set as my formatter for markdown.

```json
  "[markdown]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  }
```